tests/components/emulated_kasa/test_server.py

```diff
@@ -0,0 +1,340 @@
+"""Tests for emulated_kasa library bindings."""
+import math
+from unittest.mock import Mock, patch
```

Comment:
```suggestion
from tests.async_mock import Mock, patch
```

All new test need to use `tests.async_mock`